### ISSUE TITLE
[Bug]: Saving eps figure with usetex and no text in the plot fails

### ISSUE THREAD

**Issue by felixhorger** [matplotlib/matplotlib#25038]
### Bug summary

savefig("test.eps") fails if text.usetex = True and the current figure does not contain any text. Figure can be displayed via show().

### Code for reproduction

```python
import matplotlib.pyplot as plt

plt.rcParams["text.usetex"] = True
plt.rcParams["backend"] = "ps"
plt.ioff()

plt.figure()
# Remove labels to have no text
plt.tick_params(labelleft=False, labelbottom=False)
plt.plot(range(1, 10))
# Uncomment this to see that it's not related to the ticks
# plt.title("Hello")
plt.savefig("test.eps") # Fails
```

### Actual outcome

Traceback (most recent call last):
  File "/home/felix/plottest.py", line 11, in <module>
    plt.savefig("test.eps")
  File "/usr/lib/python3/dist-packages/matplotlib/pyplot.py", line 859, in savefig
    res = fig.savefig(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/matplotlib/figure.py", line 2311, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/usr/lib/python3/dist-packages/matplotlib/backend_bases.py", line 2210, in print_figure
    result = print_method(
  File "/usr/lib/python3/dist-packages/matplotlib/backends/backend_ps.py", line 771, in print_eps
    return self._print_ps(outfile, 'eps', *args, **kwargs)
  File "/usr/lib/python3/dist-packages/matplotlib/backends/backend_ps.py", line 808, in _print_ps
    printer(outfile, format, dpi=dpi, dsc_comments=dsc_comments,
  File "/usr/lib/python3/dist-packages/matplotlib/backend_bases.py", line 1639, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/matplotlib/cbook/deprecation.py", line 411, in wrapper
    return func(*inner_args, **inner_kwargs)
  File "/usr/lib/python3/dist-packages/matplotlib/backends/backend_ps.py", line 1074, in _print_figure_tex
    font_preamble = texmanager.get_font_preamble()
  File "/usr/lib/python3/dist-packages/matplotlib/texmanager.py", line 174, in get_font_preamble
    return self._font_preamble
AttributeError: 'TexManager' object has no attribute '_font_preamble'

### Expected outcome

Even without any text present this shouldn't fail, at least not with a cryptic error message.

### Operating system

Debian

### Matplotlib Version

3.3.4 (official Debian bullseye package) 

### Matplotlib Backend

ps

### Python version

3.9.2

### Jupyter version

N/A

### Installation

Linux package manager

**Comment by rcomer** [comment]
Thanks for the report @felixhorger.  On my machine I can reproduce this error using Matplotlib version 3.5, but not using version 3.6.  So it seems this has been fixed at some point.

**Comment by QuLogic** [comment]
This was fixed by #22422; you will have to update to 3.6.0 or later.

**Comment by felixhorger** [comment]
Thanks the both of you! :)
